packages/base/any/kernels/5.15-lts/kconfig.mk

```diff
@@ -0,0 +1,10 @@
+THIS_DIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+K_MAJOR_VERSION := 5
+K_PATCH_LEVEL := 15
+K_SUB_LEVEL := 11
```

Comment:
can we get these numbers from https://github.com/dentproject/linux/blob/dent-linux-5.15.y/Makefile#L4

Reply:
You can parse it from the cloned kernel source instead of hardcoding and having to update it.